### PR TITLE
GTEx Project, Individual, Biosample

### PR DESCRIPTION
**Pull request by buchanae**
This is skeleton code for GTEx. There are some unanswered questions:

- Does GTEx have Aliquots?
- Is there only one Project for GTEx?

### CODE REVIEW

**Comment by buchanae** [comment]
Added an Aliquot per Biosample. Sticking with one project ("gtex") for now.

This is ready for review.

File: transform/gtex/samples.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from bmeg.util.cli import default_argument_parser
+from bmeg.emitter import JSONEmitter
+from bmeg.ioutils import read_tsv
+from bmeg.vertex import Aliquot, Biosample, Project, Individual
+from bmeg.edge import AliquotFor, BiosampleFor, InProject
+
+
+parser = default_argument_parser()
```
Comment (adamstruck) [inline transform/gtex/samples.py]: Looks like this parser block could be removed.

Comment (buchanae) [reply transform/gtex/samples.py]: Fixed

**Comment by buchanae** [comment]
Ready to go. Waiting on review from @kellrott

File: transform/gtex/samples.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+from bmeg.vertex import Aliquot, Biosample, Project, Individual
+from bmeg.edge import AliquotFor, BiosampleFor, InProject
+
+
+def extract_individual_id(sample_id):
+    return "-".join(sample_id.split("-")[:2])
+
+
+emitter = JSONEmitter("gtex")
+samples = read_tsv("source/gtex/GTEx_v7_Annotations_SampleAttributesDS.txt")
```
Comment (kellrott) [inline transform/gtex/samples.py]: Not sure how I feel about hard coding the input paths in the code. At the very least the original download URLs should be documented somewhere. I'm assuming this file is coming from https://storage.googleapis.com/gtex_analysis_v7/annotations/GTEx_v7_Annotations_SubjectPhenotypesDS.txt

Comment (adamstruck) [reply transform/gtex/samples.py]: This is the pattern we are considering adopting for all transformers. Check out the style guide PR for more details https://github.com/bmeg/bmeg-etl/pull/160.

**Pull request merged**